src/sdk/object_io.js

```diff
@@ -495,6 +496,7 @@ class ObjectIO {
             });
             await mc.run();
             if (mc.had_errors) throw new Error('Upload map errors');
+            if (params.update_cache_stats()) params.update_cache_stats(params.range.end - params.range.start);
```

Comment:
The condition should not call the function like so `params.update_cache_stats()`
instead it should just check if the property exists like this `params.update_cache_stats`.

Another point - we suggested the name upload_chunk_hook here right?

So see the code suggestion, but also need to change in namespace_cache to use the new property name in the upload_params (inline or assign).

```suggestion
            if (params.upload_chunk_hook) params.upload_chunk_hook(params.range.end - params.range.start);
```

Reply:
It was `update_cache_stats_hook` so I changed it to that. And the other thing was an oversight, I corrected it